VocaDbWeb/Startup.cs

```diff
@@ -119,6 +119,33 @@ public void ConfigureServices(IServiceCollection services)
 				});
 
 			services.AddLaravelMix();
+
+			services.AddCors(options =>
+			{
+				options.AddDefaultPolicy(builder =>
+				{
+					builder
+						.AllowAnyOrigin()
+						.AllowAnyHeader()
+						.WithMethods("GET");
+				});
+
+				options.AddPolicy(AuthenticationConstants.AuthenticatedCorsApiPolicy, builder =>
+				{
+					builder
+						.AllowAnyHeader()
+						.WithMethods("GET", "POST", "PUT", "OPTIONS");

```

Comment:
Not at all versed in this - but is there a reason you set specific methods instead of all?

Reply:
Actually, we only need `GET` and `POST`.

Reply:
While all the PUT APIs are currently private, I see no reason why PUT would not be supported in the future. I also think it's ok to allow all verbs, including DELETE.

Reply:
Do you mean that we can use the [AllowAnyMethod](https://github.com/dotnet/aspnetcore/blob/c925f99cddac0df90ed0bc4a07ecda6b054a0b02/src/Middleware/CORS/src/Infrastructure/CorsPolicyBuilder.cs#L181) method, which sets `Access-Control-Allow-Methods` to `*` (wildcard)?

Reply:
Yeah, I see no issue with that.